easybuild/easyconfigs/b/Blender/Blender-2.66a-foss-2016.04-Python-3.5.2.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'CMakeMake'
+
+name = 'Blender'
+version = '2.66a'
+
+pyver = '3.5.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.blender.org/'
+description = """Blender is the free and open source 3D creation suite. It supports
+ the entirety of the 3D pipeline-modeling, rigging, animation, simulation, rendering,
+ compositing and motion tracking, even video editing and game creation."""
+
+toolchain = {'name': 'foss', 'version': '2016.04'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://download.blender.org/source/']
+
+# These are needed until extra dependencies are added for them to work 
+configopts = '-DWITH_INSTALL_PORTABLE=OFF '
+configopts += '-DWITH_BUILDINFO=OFF '
+configopts += '-DWITH_GAMEENGINE=OFF '
+configopts += '-DWITH_CYCLES=OFF '
+configopts += '-DWITH_SYSTEM_GLEW=OFF '
+
+dependencies = [
+    ('Python', pyver),
```

Comment:
@AFCJamie no need to add a `pyver` local variable, just use `'3.5.2'` here, the EasyBuild framework will make sure that the `%(pyver)s` template is resolved (see http://easybuild.readthedocs.io/en/latest/version-specific/easyconfig_templates.html)